bundles/org.openhab.binding.awattar/src/main/resources/OH-INF/i18n/awattar_de.properties

```diff
@@ -158,16 +158,15 @@ channel-group-type.awattar.tomorrow23.label = Morgen 23\:00
 channel-group-type.awattar.tomorrow23.description = Morgige Preise von 23\:00 bis 00\:00
 
 
-error.config.missing=Konfiguration fehlt\!
-error.bridge.missing=Bridge fehlt\!
-error.channelgroup.missing=Channelgroup fehlt\!
+error.config.missing=Konfiguration fehlt!
```

Comment:
Please do not edit i18n files here, edit them on CrowdIn, otherwise they'll be out of sync and cause issues. Only change the english version.